### DISCUSSION TITLE
Zoom "original sound for musicians" must be ON for Kingdom Melodies audio to be heard on Zoom.

### DISCUSSION THREAD

**Discussion by pktechnology-code** [sircharlo/meeting-media-manager#1767] [Q&A] [answered]
Anyone seeing this? 
On ver 23.4.0 if I dont turn on "original sound for musicians" then no audio for zoom participants on Kingdom Melodies. 
The strange thing is that all other videos OK, dont need to turn this setting on, just Kingdom Melodies. 
If I play the Kingdom Melody via JWL and share on Zoom, no problem, setting does not need to be on.
Is that a bug? 

thanks in advance

**Comment by mtdvlpr** [comment, marked as answer]
Hi @PK2000GIT, this is not a bug, but rather a change within Zoom. We experienced the same some time ago. See #1599 for more information.
